src/test/account.test.js

```diff
@@ -183,7 +182,7 @@ describe("Get Account by ID", () => {
       .expect(401);
   }, 25000);
 
-  test("If there is not sent an ID should return a status code 400", async () => {
+  test("If there is not sent an ID should return a status code 404", async () => {
```

Comment:
This should not return a 404 but 400 is more accurate

Reply:
The status that is returned by API is 404.